Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.6.0
+FROM node:lts
```

Comment:
It looks like `node:lts` will change to a different node version over time, which could cause code to break or cause different developers or servers to be running different code depending on when they ran `docker-compose build`. Currently `node:lts` is the same as `node:16.14.0` -- could we use that?